lib/core-generators/new/templates/views/pages/legal/privacy.ejs

```diff
@@ -38,17 +38,36 @@
 
     <h3>How you can access and update your information</h3>
     <p>Our goal is for any personal information in our possession to be as accurate, current and complete as necessary for the purposes for which we use that information. If you want to review, verify or correct your personal information, please contact us using the contact information set out below. We reserve the right not to change any personal information and instead append any alternative information the individual concerned believes is appropriate. To some extent, we rely on individuals to update their own personal information. Unless an individual advises us of these important changes, we may have no way of knowing about them.</p>
+
+    <h3>Your legal rights</h3>
     <p>The State of California enacted the Shine the Light law (CA Civil Code Sec 198.83) that permits users who are California residents to request certain information regarding the disclosure of personal information during the past year for marketing purposes. To make such a request, you can <a href="/contact">contact us</a>.</p>
+    <p>If you are in the European Union, you have rights under the General Data Protection Regulation in relation to your personal data:</p>
+    <ul>
+      <li><strong>The right to access</strong> - You have the right to request NEW_APP_COMPANY_NAME for copies of your personal data. We may charge you a small fee for this service.</li>
+      <li><strong>The right to rectification</strong> - You have the right to request that NEW_APP_COMPANY_NAME correct any information you believe is inaccurate. You also have the right to request NEW_APP_COMPANY_NAME to complete the information you believe is incomplete.</li>
+      <li><strong>The right to erasure</strong> - You have the right to request that NEW_APP_COMPANY_NAME erase your personal data, under certain conditions.</li>
+      <li><strong>The right to restrict processing</strong> - You have the right to request that NEW_APP_COMPANY_NAME restrict the processing of your personal data, under certain conditions.</li>
+      <li><strong>The right to object to processing</strong> -  You have the right to object to NEW_APP_COMPANY_NAME’s processing of your personal data, under certain conditions.</li>
+      <li><strong>The right to data portability</strong> - You have the right to request that NEW_APP_COMPANY_NAME transfer the data that we have collected to another organization, or directly to you, under certain conditions.</li>
+    </ul>
+    <p>If you make a request, we have one month to respond to you. If you would like to exercise any of these rights, please <a href="/contact">contact us</a>.</p>
+    <p>Should you wish to report a complaint or if you feel that NEW_APP_COMPANY_NAME has not addressed your concern in a satisfactory manner, you have the right to lodge a complaint with your local data protection authority.</p>
```

Comment:
Let's just say "we" instead of the name of the company again.  I know it sounds less formal and maybe it's less ideal, but I think it's worth a try to see if it makes it read a little friendlier